lib/default.nix

```
@@ -120,7 +120,9 @@ let
       noDepEntry fullDepEntry packEntry stringAfter;
     inherit (self.customisation) overrideDerivation makeOverridable
       callPackageWith callPackagesWith extendDerivation hydraJob
-      makeScope makeScopeWithSplicing makeScopeWithSplicing';
+      makeScope makeScopeWithSplicing makeScopeWithSplicing'
+      extendMkDerivation extendMkDerivationModified adaptMkDerivation adaptMkDerivationModified
```

Two of these functions aren't in evidence:
```suggestion
      extendMkDerivation adaptMkDerivation
```